RELEASE.rst

```diff
@@ -0,0 +1,28 @@
+Release
+=======
+
+This project is released on PyPI as `coveralls`_.
+
+To cut a new release, ensure the latest master passes all tests. Then, create a release commit:
+
+1. Update the :code:`CHANGELOG.rst` with the new version.
+2. Bump the version number in :code:`setup.py`.
+3. Tag that commit with the version number (:code:`git tag x.y.z`).
```

Comment:
Yes, we should not create eggs at all, eggs are deprecated stuff. We should stick to wheels and tarred sources.

>Doing all this on CI, maybe kicked off by a tag

We could, the win is minimal as this is not a common process (making releases)

Either way we should automate the changelog creation. I built a tool for that, and we are already using it in several projects. https://github.com/spyder-ide/loghub/